packages/docs/src/helpers/CodeSandboxIconsHelper.ts

```diff
@@ -46,9 +45,7 @@ export const iconsConfig = `
   const app = createApp(App)
     .use(createVuestic({
       config: {
-        {
-          icons: createIconsConfig({ fonts }),
-        }
+        icons: createIconsConfig({ fonts }),
```

Comment:
We use fontawesome in one single demo. I think we need to add a small alert to this demo that user needs to install fa font manually.

This is not related to this pr. All good here. Just noticed that here.

Reply:
We already have a notification.
![image](https://user-images.githubusercontent.com/29167241/186905450-ff4f246a-7b80-4050-90b6-013d21f4f83b.png)